src/signals/incident-management/containers/IncidentDetail/components/Detail/components/Location/index.js

```diff
@@ -83,22 +84,25 @@ const Location = ({ location }) => {
 
           {location.address_text ? (
             <div>
-              {location.stadsdeel && (
+              {configuration.useAreasInsteadOfStadsdeel && location.area_code && (
```

Comment:
That check would be getting quite complex. Firstly, it starts out as an empty array, so we should check for `districts.length`. Secondly, it actually isn't empty from the start, since it is directly filled with the option to select all districts that are not linked to a district, so we would actually need to check for `districts.length > 1` which is very ugly and fragile.

I would opt for a loading state in https://github.com/Amsterdam/signals-frontend/pull/1026/files/b2b3ed6b0ef73f1ff1e36b6c02ba87bb1a45137c#diff-b1f2b35cb13935809e5360d69a2f7706

Reply:
I will handle it in the selector so I can just check for `districts` here.